repl/parser/regen.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+ANTLR_JAR="/home/jdtatum/bin/antlr-4.10.1-complete.jar"
+antlr4="java -Xmx500M -cp \"$ANTLR_JAR:$CLASSPATH\" org.antlr.v4.Tool"
+
+$antlr4 \
+    -Dlanguage=Go \
+    -lib ../../parser/gen/ \
+    -visitor \
+    ./Types.g4 Types.g4
```

Comment:
I think you just need the one `./Types.g4` here. Also might want to make sure that `-package parser` is provided as a CLI arg.

Reply:
Done